sql/core/src/test/scala/org/apache/spark/sql/execution/command/PlanResolutionSuite.scala

```diff
@@ -160,6 +160,8 @@ class PlanResolutionSuite extends AnalysisTest {
       new ResolveCatalogs(catalogManager),
       new ResolveSessionCatalog(catalogManager, conf, _ == Seq("v"), _ => false),
       analyzer.ResolveTables,
+      analyzer.ResolveUnresolvedTableOrView,
+      new ResolveSessionCatalog(catalogManager, conf, _ == Seq("v"), _ => false),
```

Comment:
Unfortunately, we need to run `ResolveSessionCatalog` again if we want to test the resolution to `DropTableCommand`: https://github.com/apache/spark/pull/30079/files#diff-7ce7e7e03723d865798a6dcb6efa81aea25e925397e4c9288bbfd3317c8a515bR642